parm/use_cases/model_applications/convection_allowing_models/EnsembleStat_fcstHRRR_fcstOnly_SurrogateSevere.conf

```diff
@@ -69,32 +67,59 @@ ENS_VAR1_THRESH = >=14.2, >=19.0, >=26.0, >=38.0, >=61.0
 
 
 # REGRID_DATA_PLANE (Step 4)
-# run regrid_data_plane on observation data
+# Run regrid_data_plane on forecast data
 FCST_REGRID_DATA_PLANE_RUN = True
 
-# method to run regrid_data_plane, not setting this will default to NEAREST
+# If true, process each field individually and write a file for each
+# If false, run once per run time passing in all fields specified
+REGRID_DATA_PLANE_ONCE_PER_FIELD = False
+
+# Name of input field to process
+FCST_REGRID_DATA_PLANE_VAR1_INPUT_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge14.2
+FCST_REGRID_DATA_PLANE_VAR2_INPUT_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge19.0
+FCST_REGRID_DATA_PLANE_VAR3_INPUT_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge26.0
+FCST_REGRID_DATA_PLANE_VAR4_INPUT_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge38.0
+FCST_REGRID_DATA_PLANE_VAR5_INPUT_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge61.0
+
+# Level of input field to process
+FCST_REGRID_DATA_PLANE_VAR1_INPUT_LEVEL = "(*,*)"
+FCST_REGRID_DATA_PLANE_VAR2_INPUT_LEVEL = "(*,*)"
+FCST_REGRID_DATA_PLANE_VAR3_INPUT_LEVEL = "(*,*)"
+FCST_REGRID_DATA_PLANE_VAR4_INPUT_LEVEL = "(*,*)"
+FCST_REGRID_DATA_PLANE_VAR5_INPUT_LEVEL = "(*,*)"
+
+# Name of output field to create
+FCST_REGRID_DATA_PLANE_VAR1_OUTPUT_FIELD_NAME = MXUPHL_prob_75
+FCST_REGRID_DATA_PLANE_VAR2_OUTPUT_FIELD_NAME = MXUPHL_prob_80
+FCST_REGRID_DATA_PLANE_VAR3_OUTPUT_FIELD_NAME = MXUPHL_prob_85
+FCST_REGRID_DATA_PLANE_VAR4_OUTPUT_FIELD_NAME = MXUPHL_prob_90
+FCST_REGRID_DATA_PLANE_VAR5_OUTPUT_FIELD_NAME = MXUPHL_prob_95
+
+# Mask to use for regridding
+REGRID_DATA_PLANE_VERIF_GRID = G211
+
+# Method to run regrid_data_plane, not setting this will default to NEAREST
 REGRID_DATA_PLANE_METHOD = MAXGAUSS
 
-# regridding width used in regrid_data_plane, not setting this will default to 1
+# Regridding width used in regrid_data_plane, not setting this will default to 1
 REGRID_DATA_PLANE_WIDTH = 1
 
-# mask to use for regridding
-REGRID_DATA_PLANE_VERIF_GRID=G211
-
-FCST_REGRID_DATA_PLANE_VAR1_FIELD_NAME = MXUPHL_24_A1_ENS_FREQ_ge14.2
+# Set Gaussian dx value to add as command line argument - not added if unset or blank
+REGRID_DATA_PLANE_GAUSSIAN_DX = 81.271
 
+# Set Gaussian filter radius value to add as command line argument - not added if unset or blank
+REGRID_DATA_PLANE_GAUSSIAN_RADIUS = 120
 
 [dir]
-INPUT_BASE = /d3/projects/MET/METplus_Data
-OUTPUT_BASE = /d3/personal/kalb/METplus_Data
-PARM_BASE = /home/kalb/surrogate_severe/METplus/parm
+INPUT_BASE = /d1/projects/METplus/METplus_Data
```

Comment:
This setting overrides the settings in my metplus_runtime.conf and metplus_logging.conf so that when you run, the logs dir is created in METplus_Data as opposed to my personal metplus_out directory. I'm not sure how to fix this since we need data form the output directory to run EnsembleStat and RegridDataPlane.